web-console/src/views/query-view/query-view.tsx

```diff
@@ -205,14 +209,23 @@ export class QueryView extends React.PureComponent<QueryViewProps, QueryViewStat
 
         const query = QueryView.isJsonLike(queryString) ? Hjson.parse(queryString) : queryString;
 
+        const queryId = generateQueryId();
+
+        const isSql = !QueryView.isJsonLike(queryString);
```

Comment:
no need to call `QueryView.isJsonLike(queryString)` twice in 3 lines, call it once up top and then reuse the variable

Reply:
yes, i should do this. Initialed isSql variable and reuse it in query and delete api url.